Source/ASDisplayNode.mm

```diff
@@ -590,6 +595,10 @@ - (void)_locked_loadViewOrLayer
   } else {
     TIME_SCOPED(_debugTimeToCreateView);
     _view = [self _locked_viewToLoad];
+    if ([self supernode] == nil) {
+      // Move to supernode wil propagateDown other way
+        _primitiveTraitCollection = ASPrimitiveTraitCollectionFromUITraitCollection(_view.traitCollection);
```

Comment:
this will set `traitCollection` into system default value (the same as unspecified), right?

Reply:
ASDisplayNode init with userInterfaceStyle = .unspecific, and only update userInterfaceStyle when UIView call traitCollectionDidChange, it will miss UIView's default value of traitCollection.userInterfaceStyle

Reply:
Please explain why you decided to change the ivar directly instead of calling the setter (i.e `-setPrimitiveTraitCollection:`) which in turn calls `-asyncTraitCollectionDidChangeWithPreviousTraitCollection` if the values are different.